wd.sh

```diff
@@ -442,16 +442,19 @@ then
     echo "wd version $WD_VERSION"
 fi
 
+# set the config file from variable or default
+typeset wd_config_file=${WD_CONFIG:-$HOME/.warprc}
 if [[ ! -z $wd_alt_config ]]
 then
-    WD_CONFIG=$wd_alt_config[2]
+    # prefer the flag if provided
+    wd_config_file=$wd_alt_config[2]
 fi
```

Comment:
📝 Most changes in this file are focused around this new `wd_config_file` variable!